recipe/meta.yaml

```
@@ -578,6 +579,7 @@ outputs:
         - {{ pin_subpackage('libopenvino-auto-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-hetero-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-auto-batch-plugin', exact=True) }}
+        - tbb-devel  # to property find libs via openvino.pc
```

```suggestion
        - {{ pin_compatible('tbb-devel', max_pin='x.x.x') }}  # to property find libs via openvino.pc
      host:
        - tbb-devel
```
otherwise the conda-forge pinning doesn't get picked up.

---

without the `tbb-devel` in the host section, it may be "any newer tbb". The `pin_compatible` will look at what was used at build time, and pin it "more strictly", though you can decide to choose what the max pin should be. I think it should match the build time pinnings. so `x.x`